feat(setting/access-control): Enhanced dropdown list to show more information on the selected items

joseph118 opened the pull request:
1. Updated captions and removed the number placeholder for `countriesSelected` and `flavoursSelected`.
2. Updated edit-profile.component.html:
   i. Adjusted p-multiSelect to eliminate use of label with placeholder.
   ii. Adjusted p-multiSelect to make use of selectedItems template. This consists of 'the selected list' and the amount selected.
3. Updated profiles-store.service.ts to retrieve the flavor label.
4. Updated styling to adjust the height automatically and position the elements accordingly.

### PR information

**What is the current behavior?**
When selecting an item from the dropdown, it is impossible to know which items you have selected unless you re-view the dropdown list as it displays only the amount selected.
Example: `2 Countries Selected` or `1 Flavour Selected`.


**What is the new behavior?**
Display the selected items (new), and as well as the number of selected items (as before).


**Does this PR introduce a breaking change?**
No.

### Testing

Tested on Chrome, Firefox, IE11, Edge, and Safari.
![chrome](https://user-images.githubusercontent.com/6863655/72816873-1e8cd480-3c69-11ea-8c15-342e675efe98.png)
![firefox](https://user-images.githubusercontent.com/6863655/72816876-1f256b00-3c69-11ea-8512-2faf93b09521.png)
![ie11](https://user-images.githubusercontent.com/6863655/72816877-1f256b00-3c69-11ea-8e24-573b04045bcb.png)
![edge](https://user-images.githubusercontent.com/6863655/72816878-1f256b00-3c69-11ea-88a5-ad5e7447cd94.png)
![safari](https://user-images.githubusercontent.com/6863655/72816879-1fbe0180-3c69-11ea-8ca7-a16d12dfe584.png)

amirch1 commented:
@joseph118  - while I understand the problem you are trying to solve, I think that in terms of design this is problematic, especially when you have many items selected (imagine 30 countries for example). I will consult with our Graphic designer regarding that.

joseph118 commented:
@amirch1 you are right, I didn't see the problem from that point of view.
I did some adjustments on the proposed PR and added a scroll to the dropdown when there are items selected.
This will cap the visible items to 7 (can be adjusted) and will also eliminate the dropdown caret when there are items selected. 
A shadow is also added to separate the list from the amount selected.

![update-2-chrome-test](https://user-images.githubusercontent.com/6863655/72884590-ab39a000-3d06-11ea-8392-966beca5179e.png)

amirch1 commented:
@joseph118 - closing this as it wasn't approved by our UX team. Thanks for the effort!
